Units/parser-cxx.r/bug1799343-1.cpp.d/expected.tags

```diff
@@ -1,6 +1,7 @@
 C	input.cpp	/^struct C { int x; };$/;"	s	file:
-D	input.cpp	/^ D() { x = 123; }$/;"	f	struct:D
+x	input.cpp	/^struct C { int x; };$/;"	m	struct:C	file:
 D	input.cpp	/^struct D : ::C {$/;"	s	file:
+D	input.cpp	/^ D() { x = 123; }$/;"	f	struct:D	file:
```

Comment:
Again, a structure declared in a cpp file. Can't be visible from anywhere else nor can be its members.

Though nitpicking... this is again controversial since the class may be derived from another one that is actually declared in a header file and the virtual overrides *might* be actually stepped into when calling from a different file (say, in a debugger).